core/src/main/java/feign/Capability.java

```diff
@@ -137,4 +141,8 @@ default <C> AsyncContextSupplier<C> enrich(AsyncContextSupplier<C> asyncContextS
   default MethodInfoResolver enrich(MethodInfoResolver methodInfoResolver) {
     return methodInfoResolver;
   }
+
+  default <B extends BaseBuilder<B>> BaseBuilder<B> enrich(BaseBuilder<B> baseBuilder) {
```

Comment:
this doesn't make sense... the whole point of `Capability` is to change builders.... having a method that let you change the builder only indicates that the `Capability` API is incomplete.

this needs to be removed and whatever field needs customization needs to be added to the API